charts/monitoring/templates/otel-collector-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ include "common.names.fullname" . }}-otel-collector
+subjects:
+  - kind: ServiceAccount
+    name: "{{ .Release.Name }}-opentelemetry-collector"
```

Comment:
Can we stick with either `otel-collector` or `opentelemetry-collector` throughout?